_posts/2020-08-16-solution-to-bornhack-2020-nc333.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Solution to Bornhack 2020 CTF challenge nc333"
+author: capitol
+category: ctf
+---
+
+![tent_village](/images/tent_village.jpg)
+
+##### Name:
+nc333
+
+##### Category:
+crypto
+
+##### Points:
+50
+
+#### Writeup
+
+This was the second challenge in the crypto category, this time a lot of different encodings.
+
+We got [this file]({% link /assets/nc333.zip %}) that contained a large number of encodings strings
+inside each other.
```

Comment:
Might be redundant to say inside each other here? It sounds a bit off to me anyways, I just don't know a good way of phrasing it differently.